AppIntro.js

```diff
@@ -103,11 +104,14 @@ const styles = StyleSheet.create({
     justifyContent: 'center',
     alignItems: 'center',
   },
-});
+}
 
 export default class AppIntro extends Component {
   constructor(props) {
     super(props);
+
+    this.styles = StyleSheet.create(assign({}, defaulStyles, props.customStyles));
```

Comment:
the object spread operator doesn't do it deeply.

### example

```js
const one = { 
  a: {
    value: 3
  }
}

const two = { 
  a: {
    otherValue: 2
  }
}

const three = { ...one, ...two};

console.log(three);
```

The output for this will be:
```js
{
  a: {
    othervalue: 2
  }
}
```

instead of what we want in this case
```js
{
  a: {
    value: 3,
    othervalue: 2
  }
}
```